taskgraphs/nemo_examples/chatbot_example.gq.yaml

```diff
@@ -0,0 +1,168 @@
+- id: data
+  type: DialogDataLayerNode
+  conf:
+    batch_size: 128
+    corpus_name: cornell
+    datafile: notebooks/movie_data.txt
```

Comment:
Need to add instructions where to download "movie_data.txt" from. Or download it in build.sh script.

```
wget https://github.com/NVIDIA/NeMo/raw/v0.11.1/tests/data/movie_lines.txt.gz
gzip -d movie_lines.txt.gz
mv movie_lines.txt movie_data.txt
```

Reply:
done